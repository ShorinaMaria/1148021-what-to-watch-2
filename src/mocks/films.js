const films = [
  {
    title: `Fantastic Beasts: The Crimes of Grindelwald`,
    img: `fantastic-beasts-the-crimes-of-grindelwald.jpg`,
  },
  {
    title: `Pulp fiction`,
    img: `pulp-fiction.jpg`,
  },
  {
    title: `Snatch`,
    img: `snatch.jpg`,
  },
  {
    title: `Moonrise kingdom`,
    img: `moonrise-kingdom.jpg`,
  },
  {
    title: `Macbeth`,
    img: `macbeth.jpg`,
  }
];

export default films;
