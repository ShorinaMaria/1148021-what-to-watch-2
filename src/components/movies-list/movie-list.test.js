import React from 'react';
import renderer from 'react-test-renderer';
import films from '../../mocks/films';
import MoviesList from './movies-list';

it(`renders movies list correctly`, () => {
  const tree = renderer.create(
      <MoviesList
        movies={films}
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
