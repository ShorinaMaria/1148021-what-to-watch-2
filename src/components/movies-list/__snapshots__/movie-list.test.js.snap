// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders movies list correctly 1`] = `
<div
  className="catalog__movies-list"
>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <video
        height="175"
        muted={true}
        poster="img/fantastic-beasts-the-crimes-of-grindelwald.jpg"
        width="280"
      >
        <source
          src="https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm"
        />
      </video>
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
      >
        Fantastic Beasts: The Crimes of Grindelwald
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <video
        height="175"
        muted={true}
        poster="img/pulp-fiction.jpg"
        width="280"
      >
        <source
          src="https://gcs-vimeo.akamaized.net/exp=1573000301~acl=%2A%2F668495786.mp4%2A~hmac=33503a4f9f333a7e30e82b43a7e9dcb5121f22e4b50739090220c3f8c80c45c0/vimeo-prod-skyfire-std-us/01/4698/7/198491112/668495786.mp4"
        />
      </video>
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
      >
        Pulp fiction
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <video
        height="175"
        muted={true}
        poster="img/snatch.jpg"
        width="280"
      >
        <source
          src="https://gcs-vimeo.akamaized.net/exp=1573000470~acl=%2A%2F667835929.mp4%2A~hmac=37f0020dafd32b69dccd360ebaa5267890df6136ed4f2a713cac3a291a743b08/vimeo-prod-skyfire-std-us/01/4667/7/198339738/667835929.mp4"
        />
      </video>
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
      >
        Snatch
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <video
        height="175"
        muted={true}
        poster="img/moonrise-kingdom.jpg"
        width="280"
      >
        <source
          src="https://gcs-vimeo.akamaized.net/exp=1573000605~acl=%2A%2F672540334.mp4%2A~hmac=fcd0972204ec5907666c622faf03d4c357be0f0cd1a3c8ec4a8d659ceb8ea574/vimeo-prod-skyfire-std-us/01/4881/7/199406593/672540334.mp4"
        />
      </video>
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
      >
        Moonrise kingdom
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <video
        height="175"
        muted={true}
        poster="img/macbeth.jpg"
        width="280"
      >
        <source
          src="https://gcs-vimeo.akamaized.net/exp=1573000682~acl=%2A%2F675826480.mp4%2A~hmac=c11b78cc30c8c11190c0075246f35f01cf25c418f8d5f0c2e5108fd8b36f04ae/vimeo-prod-skyfire-std-us/01/32/8/200160788/675826480.mp4"
        />
      </video>
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
      >
        Macbeth
      </a>
    </h3>
  </article>
</div>
`;
